articles/sentinel/sap-solution-deploy-alternate.md

```diff
@@ -34,17 +34,28 @@ For more information, see [Azure Sentinel SAP solution detailed SAP requirements
 
 Create an Azure key vault that you can dedicate to your Azure Sentinel SAP data connector.
 
-Run the following command to create your Azure key vault:
+Run the following command to create your Azure key vault and grant access to an AZ SP : 
```

Comment:
please change to "... grant access to an Azure service principal" if that's correct?

Reply:
done